sdk/test/AzureTest/AzureTestTests/AzureTestTests.swift

```diff
@@ -0,0 +1,167 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import XCTest
+import Foundation
+import DVR
+import AzureTest
+
+class AzureTestTests: XCTestCase {
+
+    var fakeData: Data!
+    
+    var fakeRequest: URLRequest!
+    
+    var fakeResponse: URLResponse!
+    
+    var fakeResponseData: Data!
+    
+    private func chooseRecordedInteraction(number: Int) {
+        fakeRequest = fakeData.request(number: number)
+        fakeResponse = fakeData.response(number: number)
+        fakeResponseData = fakeData.responseData(number: number)
+    }
+    
+    override func setUpWithError() throws {
+        let testBundle = Bundle(for: type(of: self))
+        let path = testBundle.path(forResource: "TestData", ofType: "json")
+        fakeData = try! Data(contentsOf: URL(fileURLWithPath: path!))
+        chooseRecordedInteraction(number: 8)
+        
+    }
+
+    func test_scrubbingRequest_removeSubscriptionIDs() throws {
+        let cleanedRequest = Filter().scrubSubscriptionIDs(from: fakeRequest!)
+        let shouldPass = cleanedRequest.url?.absoluteString.contains(regex: Filter.subcriptionIDReplacement) ?? false
+        XCTAssert(shouldPass)
```

Comment:
Same as with the `test_scrubbingResponse_removeSubscriptionIDs` test, we should test that the request no longer contains the GUID, not that it does contain the replacement.

Reply:
I would also force unwrap things like `url` so you don't need `?? false`.